RUNTIME.md

```diff
@@ -0,0 +1,536 @@
+    geopmpy.plotter
+    geopmpy.update_report
+
+All other modules rely on the output of the GEOPM build (namely
+libgeopmpolicy.so) being present and available in LD_LIBRARY_PATH or
+similar.
+
+### Anaconda Usage
+To create an Anaconda environment that contains the `geopmpy` package,
+run the following commands:
+
+    conda init
+    conda activate # base env
+    conda create --name geopm
+    conda activate --stack geopm
+    conda install git pip # To ensure usage of the env's pip and not system pip
+
+    python -m pip install "git+https://github.com/geopm/geopm.git@dev#egg=geopmpy&subdirectory=scripts"
+
+BUILD REQUIREMENTS
+------------------
+In order to build the GEOPM package from source, the below
+requirements must be met.  The user can opt out of the features
+enabled by any of these requirements by providing the appropriate
+disable flag to the configure command line.
+
+The GEOPM package requires a compiler that supports the MPI 2.2 and
+C++11 standards.  These requirements can be met by using GCC version
+4.7 or greater and installing the openmpi-devel package version 1.7 or
+greater on RHEL and SLES Linux, and libopenmpi-dev on Ubuntu.
+Documentation creation including man pages further requires the
+rubygems and ruby-devel package on RHEL and SLES, or ruby and ruby-dev on
+Ubuntu.
+
+RHEL:
+
+    yum install openmpi-devel elfutils-libelf-devel ruby-devel rubygems
+
+SUSE:
+
+    zypper install openmpi-devel elfutils-libelf-devel ruby-devel rubygems
+
+UBUNTU (as of 18.04.3 LTS):
+
+    apt install libtool automake libopenmpi-dev build-essential gfortran \
+        libelf-dev ruby ruby-dev python libsqlite3-dev
+
+Requirements that can be avoided by removing features with configure
+option:
+
+   - MPI compiler: --disable-mpi
+   - A Fortran compiler:  --disable-fortran
+   - The elfutils library: --disable-ompt
+
+Alternatively these can be installed from source, and an alternate MPI
+implementation to OpenMPI can be selected (e.g. the Intel distribution
+of MPI).  See
+
+    ./configure --help
+
+for details on how to use non-standard install locations for build
+requirements through the
+
+    ./configure --with-<feature>
+
+options.
+
+
+BUILD INSTRUCTIONS
+------------------
+The source code can be rebuilt from the source RPMs available from
+OpenHPC.  To build from the git repository follow the instructions
+below.
+
+If building with the Intel toolchain the following environment variables
+must be set prior to running configure:
+
+    export CC=icc
+    export CXX=icpc
+    export FC=ifort
+    export F77=ifort
+    export MPICC=mpiicc
+    export MPICXX=mpiicpc
+    export MPIFC=mpiifort
+    export MPIF77=mpiifort
+
+To build all targets and install it in a "build/geopm" subdirectory of your
+home directory run the following commands:
+
+    ./autogen.sh
+    ./configure --prefix=$HOME/build/geopm
+    make
+    make install
+
+An RPM can be created on a RHEL or SUSE system with the
+
+    make rpm
```

Comment:
We have a whole other guide on how to build RPMs now.  We should just point to that.